include/lttng/ust-events.h

```diff
@@ -110,7 +110,12 @@ struct lttng_enum_value {
 struct lttng_enum_entry {
 	struct lttng_enum_value start, end; /* start and end are inclusive */
 	const char *string;
-	char padding[LTTNG_UST_ENUM_ENTRY_PADDING];
+	union {
+		struct {
+			unsigned int is_auto:1;
```

Comment:
works for me. A "flags" unsigned int. The option bits should be defined as e.g.

enum ...myflags {
   MYFLAG_A = (1U << 0),
}